### ISSUE TITLE
Unable to use the '^' start of line character in "Replace in File"

### ISSUE THREAD

**Issue by HopperTech** [phitranphitranphitran/regexp-saver#7]
The following replacement sort of works in "Replace in Selection" but not in "Replace in File"

{
    "label": "teststart",
    "regExp": "^test",
    "replacePattern": "test worked"
}

In Selection it will only replace the first instance.

**Comment by phitranphitranphitran** [comment]
Try adding the multiline flag?
```json
{
  "label": "teststart",
  "regExp": "^test",
  "replacePattern": "test worked",
  "regExpFlags": "gm"
}
```

**Comment by HopperTech** [comment]
Sorry,  that did not work

**Comment by HopperTech** [comment]
FYI, reviewing the code looks like it is using "flags" instead of "regExpFlags"

**Comment by phitranphitranphitran** [comment]
Thanks for the catch. Fixed this in 1.1.0. Please update and have a look!
